src/app/lib/layoutGrid/index.js

```diff
@@ -129,13 +129,13 @@ export const layoutGridItemMedium = css`
 
   grid-column: 1 / span 6;
   @media (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
-    padding: 0 ${props => (props.padding && props.padding.group2) || 'auto'};
+    ${({ padding: { group2 } = {} }) => (group2 ? `padding: 0 ${group2}` : '')};
```

Comment:
In this syntax how is the value being pulled from props?

Reply:
`${({ padding = {} }) => (padding.group2 && `padding: 0 ${padding.group2}`)};`

I think this might be easier to read, but I'm happy either way